scale_factor_utm_gui/graphic_interface/graphic_interface.py

```diff
@@ -0,0 +1,38 @@
+import tkinter as tk
+from PIL import ImageTk, Image
+
+class TkinterApp:
+    def __init__(self):
+        self.root=tk.Tk()
+
+    def create_button(self, value, function):
```

Comment:
Also, to add the doctoring

```suggestion
    def create_button(self, value: str, function: str):
        """Create button."""
```